developer-docs/configuring_sunbird/configuring_page_sections.md

```diff
@@ -50,6 +50,9 @@ Sample request for creating 'Open for enrollment' page section using Create Page
     }
 }
 ```
+
+**Note**: From Release 1.14.0 Sprint 2 onwards, the above mentioned custom attributes for open batch count are always in lowercase to align with content attributes naming convention. As such, the above mentioned page section needs to be updated to use open batch count attribute name converted to lowercase e.g. c_sunbird_dev_open_batch_count (instead of c_Sunbird_Dev_open_batch_count). Use [Update Page Section API](http://docs.sunbird.org/latest/apis/pagesapi/#operation/SectionUpdatePatch) to update any such existing page section.
```

Comment:
Release 1.14.2 onwards, the mentioned custom attributes for open batch count must be in lowercase. It aligns with content attributes naming convention. For example, the page section needs to be updated to use open batch count attribute name converted to lowercase that is **c_sunbird_dev_open_batch_count** (instead of c_Sunbird_Dev_open_batch_count). Use [Update Page Section API](http://docs.sunbird.org/latest/apis/pagesapi/#operation/SectionUpdatePatch) to convert.

Reply:
Please change the destination branch from master to 1.14.0